### ISSUE TITLE
List of Steps to Convert 23andme.txt to VCF

### ISSUE THREAD

**Issue by AOleson1** [arrogantrobot/23andme2vcf#13]
Hi,

I couldn't find any other way to contact you so I suppose it'll have to be through this. I was wondering if you could make an _exact list of steps_ of how to convert my 23andme.txt file to the VCF format using your tool. I'm asking you as a person who has near to no experience with coding or computer science. If you are able to tell me exactly how to do this, then I can share your tool with others like myself for a personal genomics class for uploading VCF files to various different programs and tools for analysis. Your assistance in this matter would be greatly appreciated. 

Thanks.

**Comment by msahasrabuddhe** [comment]
Hi,

I would also like to know whether this conversion in any way can handle
INDELs now. I have been looking for a tool or a way to map the INDELs in
23AndMe txt file while converting to VCF.

Thanks,
Mandar


--
Mandar Sahasrabuddhe
Business Head | mandar@kovidbioanalytics.com
INDIA: +91 942 168 4249 | US: +1 415 477 1017
www.kovidbioanalytics.com | +KovidBioAnalytics
<https://plus.google.com/+KovidBioAnalyticsPune/>

Kovid BioAnalytics TM
*Expanding Horizons...*

**Comment by arrogantrobot** [comment]
@AOleson1 I have updated the readme file with some more direct instructions. If you are still having trouble, please let me know. I'm currently thinking about a way to make this more user friendly.

@msahasrabuddhe there's already an issue for indel support. If you'd like to help add indel support, I'd be very appreciative. The difficulty is in finding a publicly available listing of the ref and alt alleles of the referenced indels by rsid number.

**Comment by msahasrabuddhe** [comment]
Hi Rob @arrogantrobot,

My team has been investigating and working on this issue. Here is the approach that we deviced:

1. Run the 23andme file through Plink using the --list-23-indels parameter
2. The SNPs will be processed by default in the normal process.
3. We will get the rsids list for all the indels from 23andme file as output from --list-23-indels parameter
4. Using the rsids in the output.indel file, now we can search the VCF files from 1000Genomes, example: ALL.chr22.integrated_phase1_v3.20101123.snps_indels_svs.genotypes.vcf chromosome wise indels file for the GT tag.
Issue encountered: This is a multi-sample vcf file. So which sample/genotype information should be considered for mapping to an indivuidual's data? How valid would this be?
We also searched the rsids against the Mills_and_1000G_gold_standard.indels.b37.vcf, 1000G_phase1_indels.b37.vcf and the ALL.wgs.phase1_release_v3.20101123.snps_indels_sv.sites.vcf files, but these do not have genotype information.
5. Once the multi sample issue is resolved, we will get the information for the indels in the vcf format.
6. We will then merge the snp.vcf file from plink with the indel.vcf from the above logic and then sort chromosome wise.

Let me know if this approach sounds fine. Also, any idea on the mult-sample issue in #4 and how to address that?

Thanks,
Mandar
